dspace-api/src/test/data/dspaceFolder/assetstore/orcid-works/work-277871.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
As noted in other PRs, these test data files should be moved to `src/test/resources` alongside the IT which uses them. That way they are easily findable next to the IT.